packages/runtime/src/next/nextVersion.ts

```diff
@@ -0,0 +1,5 @@
+import nextPackageJSON from 'next/package.json'
```

Comment:
Is it possible to import just the version directly so it doesn't bundle the entire package.json?

```ts
import { version } from 'next/package.json'

const [major, minor, patch] = version.split('.')

export { major, minor patch }
```

Also, maybe we should replace the `split('.')` with a regular expression at some point. Maybe have something a bit more robust haha

Reply:
Cool, that's cleaner. I think using `split('.')` is fine. If we want to be robust then we should use the `semver` package 😄 .

Reply:
Related links: 
- https://github.com/webpack/webpack/issues/13469, 
- https://stackoverflow.com/questions/64993118/error-should-not-import-the-named-export-version-imported-as-version